tests/console/check_system_info.pm

```diff
@@ -52,15 +52,33 @@ sub check_addons {
 }
 
 sub check_product {
-    my ($mypro) = @_;
-    $mypro //= uc get_var('SLE_PRODUCT', '');
+    my ($status) = @_;
+    $status //= "before";
     my $myver   = get_var('VERSION');
+    my %proname = (
+        HPC       => 'SLE-Product-HPC-' . $myver,
+        SLES      => 'SLES' . $myver,
+        SLED      => 'SLED' . $myver,
+        SLE_HPC   => 'SLE-Product-HPC-' . $myver,
+        leap      => "openSUSE-Leap",
+        Media_HPC => $myver . '-HPC',
+    );
+    my $mypro   = uc get_var('SLE_PRODUCT', '');
     my $product = '';
-    if ($mypro =~ /HPC/) {
-        $product = 'SLE-Product-HPC-' . $myver;
-    }
-    if ($mypro ne '') {
-        $product = $mypro . $myver;
+    if ($status eq 'before') {
+        if ((get_var("HDDVERSION") =~ /leap/)) {
+            $product = $proname{leap};
+        } elsif ($mypro eq "SLE_HPC") {
+            $product = $proname{SLES};
+        } else {
+            $product = $proname{$mypro};
```

Comment:
This means it is different for the product of HPC before and after migration? This is verified on SLES15SP3? Maybe this will be changed in the future.